suites/quincy/rgw/tier-1_rgw_multisite.yaml

```diff
@@ -0,0 +1,282 @@
+# Test suite for evaluating RGW multi-site deployment scenario.
+#
+# This suite deploys a single realm (India) spanning across two RHCS clusters. It has a
+# zonegroup (shared) which also spans across the clusters. There exists a master (pri)
+# and secondary (sec) zones within this group. The master zone is part of the pri
+# cluster whereas the sec zone is part of the sec datacenter (cluster).
+
+# The deployment is evaluated by running IOs across the environments.
+# conf : conf/pacific/rgw/rgw_multisite.yaml
```

Comment:
Please try to add a quincy specific conf file

Reply:
made changes as per comment